HelpSource/Classes/List.schelp

```diff
@@ -6,16 +6,36 @@ categories::Collections>Ordered
 DESCRIPTION::
 List is a subclass of SequenceableCollection with unlimited growth in size. Although not a subclass of link::Classes/Array:: or its superclass link::Classes/ArrayedCollection:: it uses an Array in its implementation and is in many cases interchangeable with one. (List implements many of the same methods as Array.)
 
-Arrays have a fixed maximum size. If you add beyond that size a new Array is created and returned, but you must use an assignment statement or the new array will be lost. (See the link::Classes/Array:: helpfile.) List has no size limitation and is thus more flexible, but has slightly more overhead.
+note::
+Size limitations of arrays and flexibility of lists in SuperCollider
+
+Arrays have a fixed maximum size. If you add beyond that size, a new link::Classes/Array:: is created and returned, but you must use an assignment statement or the new array will be lost. (See the link::Classes/Array#description#Description:: section in link::Classes/Array::. documentation). The idiomatic way to add new item in link::Classes/Array:: is:
```

Comment:
The text would read better like this (adjust to the formatting decision):

Arrays have a fixed maximum size. Adding beyond that size, a new link::Classes/Array:: is created and returned. In order to be able to use it, the variable holding the old array can be reassigned to the new one. Threfore, the idiomatic way to add new item in link::Classes/Array:: is:

Reply:
Thanks, done in https://github.com/supercollider/supercollider/pull/6536/commits/87bea3eeaeb0ab3b12fec0958b35647a1249ff34!